middleware/cmd/middleware/main.go

```diff
@@ -28,6 +28,7 @@ func main() {
 	bbbCmdScript := flag.String("bbbcmdscript", "/opt/shift/scripts/bbb-cmd.sh", "Path to the bbb-cmd file that allows executing system commands")
 	prometheusURL := flag.String("prometheusurl", "http://localhost:9090", "Url of the prometheus server in the form of 'http://localhost:9090'")
 	redisPort := flag.String("redisport", "6379", "Port of the Redis server")
+	redisMock := flag.String("redismock", "false", "Mock redis for development instead of connecting to a redis server, default is 'false', use 'true' as an argument to mock")
```

Comment:
You can actually set a flag to be a bool by default, not a string
`redisMock := flag.String("redismock", false, ...`
So then you should be able to get rid of this whole block:
```
and just use
`middleware, err := middleware.NewMiddleware(argumentMap, *redisMock)`
	mockRedisFlag := false
	if *redisMock == "true" {
		mockRedisFlag = true
	}
```

Reply:
Nice :)

Reply:
Ok, changed it.